src/routes/PrimaryLayout/components/Navigation/Navigation.js

```diff
@@ -65,22 +74,28 @@ export default function Navigation (props) {
     }
   ])
 
+  if (!isCommunityPath(location.pathname)) links.shift()
+
   const collapserState = collapsed ? 'collapser-collapsed' : 'collapser'
 
+  const canView = !community || community.memberCount !== 0
```

Comment:
why would canView be true if !community?

Reply:
The canView refers to the Navigation component and it visible in network, all, and public contexts where there is no community.